testgrid/config.yaml

```diff
@@ -6366,6 +6374,15 @@ dashboards:
     test_group_name: pull-cloud-provider-aws-test
     description: aws cloud provider tests
 
+- name: sig-alibaba-cloud-provider-alibaba
```

Comment:
The `sig-foo` naming scheme is reserved for sigs as listed in https://github.com/kubernetes/community

I would recommend renaming this to `cloud-provider-alibaba`, then adding a new `sig-cloud-provider` `dashboard_group` below that includes this

Reply:
I will update according to your suggestion.